### PR TITLE
New version: Google.Chrome.Canary version 124.0.6323.0

### PR DESCRIPTION
**Pull request by SpecterShell**
Created by [🥟 Dumplings](https://github.com/SpecterShell/Dumplings).
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/140986)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978335226 2024-02-26 03:43:07.548 [CLI ] Installer [X86,exe,User,] not applicable: Installer scope does not match required scope: User != Machine 2024-02-26 03:43:07.548 [CLI ] Installer [X64,exe,User,] not applicable: Installer scope does not match required scope: User != Machine 2024-02-26 03:43:30.274 [CLI ] ShellExecute installer failed: 33 2024-02-26 03:43:30.276 [CLI ] Terminating context: 0x8a150006 at D:\a\_work\1\s\external\pkg\src\AppInstallerCLICore\Workflows\InstallFlow.cpp:1fd [0226/034330.085:ERROR:persistent_histogram_storage.cc(121)] Could not write "SetupMetrics" persistent histograms to file as the storage directory does not exist. 2024-02-26 03:43:20.301 [CLI ] Installer [X86,exe,User,] not applicable: Installer scope does not match required scope: User != Machine 2024-02-26 03:43:20.301 [CLI ] Installer [X64,exe,User,] not applicable: Installer scope does not match required scope: User != Machine 2024-02-26 03:43:20.301 [CLI ] Installer [X64,exe,User,] not applicable: Machine is not compatible with X64 2024-02-26 03:43:20.301 [CLI ] Installer [X64,exe,Machine,] not applicable: Machine is not compatible with X64 2024-02-26 03:43:43.030 [CLI ] ShellExecute installer failed: 33 2024-02-26 03:43:43.033 [CLI ] Terminating context: 0x8a150006 at D:\a\_work\1\s\external\pkg\src\AppInstallerCLICore\Workflows\InstallFlow.cpp:1fd [0226/034342.854:ERROR:persistent_histogram_storage.cc(121)] Could not write "SetupMetrics" persistent histograms to file as the storage directory does not exist.

(Automated response - build 813.)

**Review by stephengillie — COMMENTED** [review]
These sections look valid - why are they commented?

File: manifests/g/Google/Chrome/Canary/124.0.6323.0/Google.Chrome.Canary.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  InstallerSwitches:
+    Custom: --chrome-sxs --do-not-launch-chrome
+  ProductCode: Google Chrome SxS
+# - Architecture: x86
+#   Scope: machine
+#   InstallerUrl: https://dl.google.com/release2/chrome/adarmluiggvp272qjxihafqeztcq_124.0.6323.0/124.0.6323.0_chrome_installer.exe
+#   InstallerSha256: C420D5B0AB1BC5FBAF74A2F691F8495C4E1466919DAC61E1B6C5DC053B35E874
+#   InstallerSwitches:
+#     Custom: --chrome-sxs --do-not-launch-chrome --system-level
+#   ProductCode: Google Chrome SxS
```
Comment (stephengillie) [inline manifests/g/Google/Chrome/Canary/124.0.6323.0/Google.Chrome.Canary.installer.yaml:49]:
```suggestion
 - Architecture: x86
   Scope: machine
   InstallerUrl: https://dl.google.com/release2/chrome/adarmluiggvp272qjxihafqeztcq_124.0.6323.0/124.0.6323.0_chrome_installer.exe
   InstallerSha256: C420D5B0AB1BC5FBAF74A2F691F8495C4E1466919DAC61E1B6C5DC053B35E874
   InstallerSwitches:
     Custom: --chrome-sxs --do-not-launch-chrome --system-level
   ProductCode: Google Chrome SxS
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
This section and the other look valid. Why are they commented?

File: manifests/g/Google/Chrome/Canary/124.0.6323.0/Google.Chrome.Canary.installer.yaml (hunk 1 above)
Comment (SpecterShell) [reply manifests/g/Google/Chrome/Canary/124.0.6323.0/Google.Chrome.Canary.installer.yaml:49]: @stephengille There are some issues with the `--system-level` switch which blocked installation and caused previous validations failed.

Comment (SpecterShell) [reply manifests/g/Google/Chrome/Canary/124.0.6323.0/Google.Chrome.Canary.installer.yaml]: Turns out that Chrome Canary doesn't support installing in machine scope by using the `--system-level` switch. The entries have been removed now.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**